### DISCUSSION TITLE
How to save the speech generated as an audio file on the device

### DISCUSSION THREAD

**Discussion by abhineetsharmathegreat** [ak1394/react-native-tts#179] [Q&A]
I have a problem that I have to use the speech generated by this library into react-native-push-notification and I wasn't able to pass the speak function as a soundName in the library. Is there some sort of way to save the speech generated so as to pass it as an audio file in the soundName property.
Thanks and Regard

**Comment by sixman9** [comment]
I would also like this feature, across IOS/Android, I would be willing to offer a figure in the low hundreds of dollars to make this happen, it would be a VERY useful feature.
